tests/album-create.test.js

```diff
@@ -0,0 +1,57 @@
+// tests/album-create
+
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+  let db;
+  let artistEntryId;
+
+  beforeEach(async () => {
+    db = await getDb();
+    //create artist - required for album creation
```

Comment:
The comments throughout here aren't really necessary as the code is nice and clear about what it is doing already